backend/compact-connect/stacks/ingest_stack.py

```diff
@@ -36,7 +36,7 @@ def __init__(
             encryption=QueueEncryption.KMS,
             encryption_master_key=persistent_stack.shared_encryption_key,
             enforce_ssl=True,
-            retention_period=Duration.hours(2),
+            retention_period=Duration.hours(12),
```

Comment:
After some testing, I got a better feel for how burning the queue off performs. While it is unlikely, it is possible for backlog to cause messages on the queue to be at risk of aging off before they are processed. In the future, we will be setting an alarm based on the age of the oldest message being over half the retention period, so that someone can investigate the unusual behavior. I would rather have that person have six hours to figure out what is going on and resolve it than only one, before messages start falling onto the dead letter queue.

Reply:
Makes great sense and seems like a smart approach, thank you.